lychee.toml

```diff
@@ -0,0 +1,72 @@
+#############################  Disclaimer  #############################
+
+# Configuration file for lychee link checker.
+#
+# Further information and explanation of possible options can be found
+# at https://github.com/lycheeverse/lychee/.
+#
+# Adapted from example config file: https://github.com/lycheeverse/lychee/blob/master/lychee.example.toml
+
+#############################  Display  #############################
+
+# Verbose program output
+verbose = true
+
+#############################  Runtime  #############################
+
+# Maximum number of concurrent link checks.
+max_concurrency = 4
+
+#############################  Requests  ############################
+
+# Comma-separated list of accepted status codes for valid links.
+accept = [200, 429]
+
+# Remap URI matching pattern to different URI.
+remap = [
+    'file:///home/runner/work/music-encoding.github.io/music-encoding.github.io/_tutorials https://github.com/music-encoding/music-encoding.github.io/tree/gh-pages/tutorials/',
+    'https://github.com/music-encoding/music-encoding/tree/basic https://github.com/music-encoding/music-encoding/commit/b50748ce537b43e1527791fb41e8c28aa1582cdb',
+    'http://music-encoding.org/community/conference/tours17/program/ https://web.archive.org/web/20170511140944/http://music-encoding.org/community/conference/tours17/program',
+    'http://music-encoding.org/community/mei-organization/mei-by-laws/ https://music-encoding.org/community/mei-by-laws.html',
+    'http://music-encoding.org/proceedings https://github.com/music-encoding/mec-proceedings',
+]
+
+#############################  Exclusions  ##########################
+
+# Exclude these URLs from checking (supports regex).
+exclude = [ 
+    # custom User files
+    'file:///Users/*',
+
+    # these DOIs are not correctly resolved (probably too many redirects)
+    'https://doi.org/10.13140/RG.2.2.15014.93760',
+    'https://doi.org/10.15463/music-1',
+
+    # page outdated, but used in candidate statements
+    'https://arsmusicae.org',    
+]
+
+# Exclude these filesystem paths from getting checked.
+exclude_path = [
+    # folders and pages with Jekyll liquids
+    '_includes',
+    '_layouts',
+    'index.html',
+    'community/projects-users.md',
+    'conference/past.md',
+
+    # Jekyll's site folder
+    '_site',
+    
+    # past conferences may contain outdated links
+    '_conferences/2018',
+    '_conferences/2019',
+    '_conferences/2020',
+    '_conferences/2021',
+]
```

Comment:
```suggestion

    # past election candidate statements may contain outdated links
    'community/mei-board/elections/2018/candidates.md',
    'community/mei-board/elections/2019/candidates.md',
    'community/mei-board/elections/2020/candidates.md',
    'community/mei-board/elections/2021/candidates.md'
]
```

Reply:
good point and in sync with your comment above (cf. https://github.com/music-encoding/music-encoding.github.io/pull/412#discussion_r1042716885)